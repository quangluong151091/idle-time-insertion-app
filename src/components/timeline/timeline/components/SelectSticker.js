import React  from 'react';

class SelectSticker extends React.Component {
    render(){
        return(
            <select id="stickerSelected" name="stickerSelected" hidden>
                <option value="" hidden>Select a sticker to add</option>
                <optgroup label="MAINTENANCE">
                  <option value="enduser" label="End User">End User</option>
                  <option value="primapower" label="Prima Power">Prima Power</option>
                </optgroup>
                <optgroup label="SETUP">
                  <option value="material" label="Material">Material</option>
                  <option value="tool" label="Tool">Tool</option>
                </optgroup>
                <optgroup label="FAILURE">
                  <option value="ic-icon" label="Operation">Operation</option>
                  <option value="ic-icon" label="Tool">Tool</option>
                  <option value="ic-icon" label="Material">Material</option>
                  <option value="ic-icon" label="Production Order">Production Order</option>
                  <option value="ic-icon" label="Program">Program</option>
                </optgroup>
                <optgroup label="Other">
                  <option value="ic-icon" label="Other">Other</option>
                </optgroup>
              </select>
        );
    }
}

export default SelectSticker;
