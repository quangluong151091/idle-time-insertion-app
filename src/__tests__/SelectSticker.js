import React from 'react';
import { shallow } from 'enzyme';

import SelectSticker from '../components/timeline/timeline/components/SelectSticker';

describe('SelectSticker', () => {
  it('should render correctly', () => {

    const component = shallow(<SelectSticker />);
  
    expect(component).toMatchSnapshot();
  });
});