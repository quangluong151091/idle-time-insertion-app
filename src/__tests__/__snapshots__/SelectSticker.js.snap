// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectSticker should render correctly 1`] = `
<select
  hidden={true}
  id="stickerSelected"
  name="stickerSelected"
>
  <option
    hidden={true}
    value=""
  >
    Select a sticker to add
  </option>
  <optgroup
    label="MAINTENANCE"
  >
    <option
      label="End User"
      value="enduser"
    >
      End User
    </option>
    <option
      label="Prima Power"
      value="primapower"
    >
      Prima Power
    </option>
  </optgroup>
  <optgroup
    label="SETUP"
  >
    <option
      label="Material"
      value="material"
    >
      Material
    </option>
    <option
      label="Tool"
      value="tool"
    >
      Tool
    </option>
  </optgroup>
  <optgroup
    label="FAILURE"
  >
    <option
      label="Operation"
      value="ic-icon"
    >
      Operation
    </option>
    <option
      label="Tool"
      value="ic-icon"
    >
      Tool
    </option>
    <option
      label="Material"
      value="ic-icon"
    >
      Material
    </option>
    <option
      label="Production Order"
      value="ic-icon"
    >
      Production Order
    </option>
    <option
      label="Program"
      value="ic-icon"
    >
      Program
    </option>
  </optgroup>
  <optgroup
    label="Other"
  >
    <option
      label="Other"
      value="ic-icon"
    >
      Other
    </option>
  </optgroup>
</select>
`;
